ow_lander/scripts/path_planning_commander.py

```diff
@@ -194,8 +205,29 @@ def run(self):
     self.guarded_move_srv = rospy.Service(
         'arm/guarded_move', GuardedMove, self.handle_guarded_move)
     print("path_planning_commander has started!")
+
     rospy.spin()
 
+  def stop_arm(self):
+    plan = activity_guarded_move.stop_guarded_move_plan(self.arm_move_group)
+    self.trajectory_async_executer.systemFaultListener(plan.joint_trajectory, self.handle_guarded_move_feedback)
```

Comment:
This function is never referenced and does seems valid, i think it needs to be dropped
```suggestion
```